home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/polls/QueryPoll.java

```diff
@@ -70,7 +70,6 @@ void getAllPollsAscSort() throws ApiException {
     void getAllPollsDescSort() throws ApiException {
 
         createPoll();
-        createPoll();
```

Comment:
Why it was removed?

Reply:
I decided that in the previous tests we are already creating a number of polls. And that's enough. Maybe I was wrong.